bin/yb-ctl

```diff
@@ -1348,23 +1365,33 @@ class ClusterControl:
         if not retry_call_with_timeout(call_change_config, self.options.timeout):
             raise ExitWithError("Could not change master config...")
 
+    def keep_and_dump_log(self, log_path):
```

Comment:
nit: can we rename this to just `dump_file_to_stderr`, since having a `keep_` prefix makes me think we would also have an API that removes it.